client/app/scripts/superdesk-users/views/user-preferences.html

```diff
@@ -26,7 +26,7 @@
                 <div class="legend" translate>Article Defaults</div>
 
                 <div class="legend" translate>Dateline</div>
-                <div sd-info-item>
+                <div sd-info-item ng-if="preferencesLoaded === true">

```

Comment:
Agreed. But I wanted to be explicit here as even a non-null value evaluates to true.